Editor/UI/EditorWindows/AvatarLoaderEditor/AvatarLoaderEditor.cs

```diff
@@ -28,6 +28,9 @@ public class AvatarLoaderEditor : EditorWindow
         [MenuItem("Ready Player Me/Avatar Loader", priority = 0)]
         public static void ShowWindow()
         {
+#if !GLTFAST
+            ModuleInstaller.AddGltfastSymbol();
+#endif
```

Comment:
This feels super hacky, what if someone doesn't open this window at all?

Reply:
So the only case where a recompile wont happen before an avatar is loaded is if they open the Avatar loader Window. 
For example: 
- If they want to run a sample, they need to import the sample (which will trigger a recompile). 
- if the restart the editor, that will also trigger a recompile
- if they add a script to start using avatar loader it will trigger a recompile. 
BUT if they open avatar loader window and try load from there it would not, so the newly added symbol won't be recognized unless I do the above ^ 

But to your point I agree it is not the best way :D More than open to ideas though.

Reply:
Makes perfect sense, i don't get why `CompilationPipeline.RequestScriptCompilation` doesn't work. But anyway this should be fine.